pkg/openscap/openscap.go

```diff
@@ -220,3 +222,26 @@ func (s *defaultOSCAPScanner) ResultsFileName() string {
 func (s *defaultOSCAPScanner) HTMLResultsFileName() string {
 	return path.Join(s.ResultsDir, HTMLResultFile)
 }
+
+func ParseResults(report []byte) []iiapi.Result {
+	ret := []iiapi.Result{}
+	node := xmldom.Must(xmldom.ParseXML(string(report))).Root
+	for _, c := range node.Query("//rule-result") {
+		if !strings.Contains(c.GetChild("result").Text, "fail") {
+			continue
+		}
+		result := iiapi.Result{
+			Name:           "openscap",
+			ScannerVersion: "1.2.3", // FIXME
```

Comment:
@enoodle actually the reason I didn't added it was this:

```
[@dev] .../openshift/clamav # oscap -q -V | wc -l
92
```

i really hate to parse the version out. how about setting this to '1.2' and making it a const for now?